src/schema/v2/partner.ts

```diff
@@ -200,7 +204,11 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
             has_published_artworks: args.hasPublishedArtworks,
           }
 
-          return allViaLoader(partnerArtistsForPartnerLoader, {
+          // If we are authenticated / loader exists use the all loader
+          const partnerArtistsLoader =
+            folioPartnerArtistsAllLoader ?? partnerArtistsForPartnerLoader
```

Comment:
this is quite weird and I am hoping there is a better way to do this:

the field on partner artist, artworks_count that is needed to populated the managedArtworks count field I have added here is not returned unless I explicitly request all fields from gravity which is the reason for the other loader:
`partner/${id}/partner_artists/all`

Reply:
It's because `artworks_count` is only included when [requesting `:all` properties](https://github.com/artsy/gravity/blob/db8d902085f39a13d2a888611aeddbe35d16bffa/app/models/domain/partner_artist.rb#L66), which is what the `/all` endpoint [requests](https://github.com/artsy/gravity/blob/db8d902085f39a13d2a888611aeddbe35d16bffa/app/api/v1/partners_artists_endpoint.rb#L121) (but not the other public one). I think the reason is that the count includes unpublished artworks.

Reply:
yeah you are right. I made this an explicit param `includeAllFields` to keep it backwards compatible, let me know what you think!